### ISSUE TITLE
Default dates for ticket types and pricing can be more useful

### ISSUE THREAD

**Issue by mikethetike** [big-neon/bn-web#328]
When entering dates for ticket types, the date needs to be edited each time. Maybe we can set the on sale time to now, and the off sale time to midnight on the event day. That will result in less clicks to create an event

**Comment by Jasonvdb** [comment]
Yeah something @zmelamed mentioned as well. We might have everything set by default with a 'advanced' button to expand and set those details manually. Waiting on design updates for that.
